README.md

```diff
@@ -235,6 +235,8 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Cloudmersive](https://cloudmersive.com/) — Utility API platform with full access to expansive API Library including Document Conversion, Virus Scanning, and more with 800 calls/month.
   * [Colaboratory](https://colab.research.google.com) — Free web-based Python notebook environment with Nvidia Tesla K80 GPU.
   * [Collect2](https://collect2.com) — Create an API endpoint to test, automate, and connect webhooks. The free plan allows for two datasets, 2000 records, 1 forwarder, and 1 alert.
+  * [CometLLM](https://github.com/comet-ml/comet-llm) - Log, visualize, organize, and search your prompt, prompt templates, prompt variables, prompt chains and other metadata in one convenient UI. We're now open-source!
```

Comment:
Please read the top of the README to find out what services are suiotable for the list and then update your entries

Reply:
Hi @ripienaar! Thanks for the feedback. I believe I addressed all the issues, but I'll admit I'm not 100% sure. If you could let me know if there are any other issues, I'd be grateful & very happy to update :) Thank you!